site/en/docs/extensions/mv3/manifest/oauth2/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Manifest - oauth2"
+date: 2022-11-14
+updated: 
+description: Reference documentation for the oauth2 property of manifest.json.
+---
+
+An optional manifest key enabling the use of an oauth2 security ID on the extension. Implementing this manifest key will require the use of the sub-properties "client_id" and "scopes", as well as the "key" property. For more detailed implementation instructions, visit the full [oauth2 tutorial](/docs/extensions/mv3/tut_oauth/).
+ ```json
+{
+  // ...
+     "oauth2": {
+      "client_id": "yourExtensionOAuthClientIDWillGoHere.apps.googleusercontent.com",
```

Comment:
Since the value we have in this key is a placeholder, the name should use SCREAMING_SNAKE_CASE. 

https://developer.chrome.com/docs/handbook/extensions/#placeholder

```suggestion
      "client_id": "YOUR_EXTENSION_OAUTH_CLIENT_ID.apps.googleusercontent.com",
```

Per the [Google developer documentation style guide](https://developers.google.com/style/placeholders#explain-placeholders), we should follow up this code block with a brief description of what `YOUR_EXTENSION_OAUTH_CLIENT_ID` is referencing or link out to relevant docs.